### ISSUE TITLE
508-defect-3 [COGNITION]: Labels SHOULD be explicit and clear for children who have left the household in the Review page

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#13016]
## [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

While working through multiple use cases, I removed a child from the household. When I got to the Review screen, I noticed there were implicit labels for the child's guardian. I'd like to improve these labels so users ensure they are submitting accurate data. Screenshot attached below.

/cc @peggygannon for content awareness

## Possible Solution
- For example: Change this to "Parent or Guardian's first name"

## Tasks
- [x] Update field names

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [x] As a user, I want to be sure the data I am submitting is correct
- [x] As a user, I want to understand what each data point represents, without seeing duplicate labels close to one another

## WCAG or Vendor Guidance (optional)

* [Error Prevention (Legal, Financial, Data): Understanding SC 3.3.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-reversible.html)

## Screenshots or Trace Logs

![Screen Shot 2020-08-28 at 10 21 58 AM](https://user-images.githubusercontent.com/934879/91617327-671e1600-e94d-11ea-8d84-5f583418e49e.png)

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**Comment by jason-gcio** [comment]
I think we understand that this change would result in:

**First name** being changed to 
**First name of parent or legal guardian**

...which will have to be edited on the form itself in order to appear here.  I don't recall any of the research showing this to be an issue, but it may still be argued that it will affect some if not clear who the name is at this point.  Is having the extra long label name on the form ok?  I am concerned that the label would be overly long or verbose.  Does this apply to country and state, etc as well?

**Comment by 1Copenut** [comment]
@allison0034 & @emilywaggoner Jason brings up a good point here, that the form labels will need to be much longer in order to have those longer labels appear on the review screen. I'm good at pointing out the ambiguity in the labels, but can't offer much guidance on label length in forms. Do either of you have suggestions? I'm okay tabling this as an enhancement if I'm inventing an issue.

**Comment by allison0034** [comment]
I looked at design.va.gov and I dont see one specific for guardian. I do see a label for spouse is "Spouse’s first name". Would "Parent's/Guardian's first name" be ok?

I do not know of any issues in having a long label except space and its takes longer for screen readers to read.

**Comment by jason-gcio** [comment]
I don't think we _need_ to include the "legal" part, this is just to make sure we are mapping the right fields to the user.  I really like the "**Parent's/Guardian's first name**" suggestion and would like to try it on for size and see how it goes.  (It helps clarify and is also not as long as I thought.)

This form is long and winding and I think I recall a few other points where this sort of specificity might be a good idea to noodle more on.

**Comment by jason-gcio** [comment]
Could also be "**Parent or Guardian's first name** as I read it back to myself...

**Comment by peggygannon** [comment]
@jason-gcio I would recommend this for the label (removing the / for readability) 
Parent or guardian's first name

**Comment by jason-gcio** [comment]
Attaching Josh for visibility
